### PR TITLE
Fix remediation ubtu 20 010065

### PR DESCRIPTION
**Pull request by dexterle**
#### Description:

- Fix UBTU-20-010065
- Fix Ansible remediation `smartcard_configure_cert_checking` and use proper styling

#### Rationale:

- Part of Ubuntu 20.04 DISA STIG v1r9 profile upgrade
- https://github.com/ComplianceAsCode/content/pull/10738

#### Review Hints:

Build the product:
```
./build_product ubuntu2004
```
To test these changes with Ansible:
```
ansible-playbook build/ansible/ubuntu2004-playbook-stig.yml --tags "DISA-STIG-UBTU-20-010065"
```

Checkout Manual STIG OVAL definitions, and use software like DISA STIG Viewer to view definitions.
```
git checkout dexterle:add-manual-stig-ubtu-20-v1r9
```

This STIG can be tested with the latest Ubuntu 2004 Benchmark SCAP. For reference, please review the latest artifacts: https://public.cyber.mil/stigs/downloads/

### CODE REVIEW

File: linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/smartcard_configure_cert_checking/ansible/shared.yml [hunk 1]
```diff
@@ -4,19 +4,49 @@
 # complexity = low
```
Comment (dexterle) [inline linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/smartcard_configure_cert_checking/ansible/shared.yml:4]: Given https://github.com/ComplianceAsCode/content/commit/e7d99c03246ebc5eacba43156987e3548a6ea73e

Please ensure that `multi_platform_ubuntu` is added

Comment (dodys) [reply linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/smartcard_configure_cert_checking/ansible/shared.yml:4]: I think you could consolidate also the bash remediation into a single shared file

**Review by dodys — CHANGES_REQUESTED** [review]

File: linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/smartcard_configure_cert_checking/ansible/shared.yml (hunk 1 above)
Comment (dexterle) [reply linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/smartcard_configure_cert_checking/ansible/shared.yml:4]: thank you

File: linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/smartcard_configure_cert_checking/bash/shared.sh [hunk 2]
```diff
@@ -1,6 +1,10 @@
-# platform = Red Hat Enterprise Linux 7,Red Hat Enterprise Linux 8,multi_platform_ol,multi_platform_sle
+# platform = Red Hat Enterprise Linux 7,Red Hat Enterprise Linux 8,multi_platform_ol,multi_platform_sle,multi_platform_ubuntu
 
+{{% if 'ubuntu' in product %}}
+{{{ bash_package_install("libpam-pkcs11") }}}
+{{% else %}}
 {{{ bash_package_install("pam_pkcs11") }}}
+{{% endif %}}
```
Comment (dodys) [inline linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/smartcard_configure_cert_checking/bash/shared.sh:8]: you should consider the case where pam_pkcs11.conf does not exist

**Review by dodys — CHANGES_REQUESTED** [review]

**Comment by dodys** [comment]
dexterle reached out last week about continuing work on this PRs, and I've asked to open new ones for a clean state. Therefore closing this one.

**Pull request closed without merging**